cpp/modmesh/view/RPythonConsoleDockWidget.cpp

```diff
@@ -112,13 +124,112 @@ void RPythonConsoleDockWidget::executeCommand()
 {
     std::string const code = m_command_edit->toPlainText().toStdString();
     appendPastCommand(code);
-    m_history_edit->insertPlainText(m_command_edit->toPlainText());
-    m_history_edit->insertPlainText("\n");
+    printCommandHistory();
     m_command_edit->setPlainText("");
     m_command_string = "";
     m_current_command_index = static_cast<int>(m_past_command_strings.size());
     auto & interp = modmesh::python::Interpreter::instance();
-    interp.exec_code(code);
+
+#ifdef _MSC_VER
+    FILE *stdoutFile, *stderrFile;
+
+    errno_t err = fopen_s(&stdoutFile, "stdout.txt", "wb");
```

Comment:
Writing to disks incurs unwanted runtime overhead.  I do not understand why do you use the file names.  Could you elaborate?

Reply:
I just want to get file descriptor. indeed we can use `pipe()`, but more work to support both Windows and Unix.

Reply:
I think it is OK to focus on one system and then the other.